index.csv

```diff
@@ -225,3 +225,6 @@ longest,MIT,1.0.1
 repeat-element,MIT,1.1.2
 trim,MIT,0.0.1
 css-value,MIT,0.0.1
+rework,MIT,1.0.1
+process,MIT,0.5.2
+exif-parser,MIT,0.1.12
```

Comment:
this also applies to v0.1.11, v0.1.10, v0.1.9, v0.1.8, v0.1.7, v0.1.6, v0.1.5, v0.1.4, v0.1.3, v0.1.2, v0.1.1, and v0.1.0 - can you add those entries as well?